include/inviwo/core/io/serialization/serializebase.h

```diff
@@ -171,46 +164,32 @@ class IVW_CORE_API SerializeBase {
 
 namespace detail {
 
+IVW_CORE_API void numericalFromStr(std::string_view value, double& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, float& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, char& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, signed char& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, unsigned char& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, short& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, unsigned short& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, int& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, unsigned int& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, long& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, unsigned long& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, long long& dest);
+IVW_CORE_API void numericalFromStr(std::string_view value, unsigned long long& dest);
+
 template <class T>
 decltype(auto) toStr(const T& value) {
-    if constexpr (std::is_same_v<std::string, T>) {
-        return value;
-    } else if constexpr (std::is_same_v<std::string_view, T>) {
-        return std::string{value};
```

Comment:
We want to still keep these ones... no point in passing a string through fmt::format